writenow

```diff
@@ -107,21 +121,20 @@ function write_lines() {
   for row in "${ROW_LINES[@]}"; do
     echo "${row}" >> "$1"
   done
+  # nice to have newline separating this entry and the next
+  echo "" >> "$1"
 }
 
 # Handle termination signals
-# Globals: TEMINATED
+# Globals: OUTPATH
```

Comment:
What if I don't want to write my output to a file and want to just discard? the exit functionality is broken here.
![Screenshot from 2025-01-14 15-30-15](https://github.com/user-attachments/assets/efb8e300-d2e3-40a1-b4b5-9f1417b84a93)